.build/build.ps1

```diff
@@ -39,6 +64,32 @@ task Publish.Module Build.Module, {
     {
         throw 'Missing required parameter "Version".'
     }
+
+    if (-not $ApiKey)
+    {
+        throw 'Missing required parameter "ApiKey".'
+    }
+
+    try
+    {
+        Import-Module -Name $modulePath -ErrorAction 'Stop'
+        $moduleInfo = Get-Module -Name $moduleName -ErrorAction 'Stop'
+    }
+    catch
+    {
+        throw $_
+    }
+
+    $parameters = @{}
+    foreach ($key in @('Tags', 'IconUri', 'ProjectUri', 'LicenseUri', 'ReleaseNotes'))
+    {
+        if ($moduleInfo.$key)
+        {
+            $parameters.Add($key, $moduleInfo.$key)
+        }
+    }
+
+    Publish-Module -Name $moduleName -Repository 'PSGallery' -NuGetApiKey $ApiKey -Force -AllowPrerelease @parameters
 }
 
 task Test.Unit PreRequisites, {
```

Comment:
I know this isn't this PR but shouldn't this be `Test.PowerShell`? `Test.Meta` should probably be `Test.PowerShellMetadata` or something. I'm not sure what `Meta` means.

As a side note, should we split this file into separate ones per solution?